lib/index.js

```diff
@@ -39,7 +39,13 @@ internals.implementation = function (server, options) {
       // otherwise use the same key (String) to validate all JWTs
       var decoded;
       try {
-        decoded = JWT.decode(token); // decode is non-io and fast enough to not have to be async
+		if (options.completeToken === true) { 
```

Comment:
@cvillerm setting complete to `true` by default does not *break* backwards compatibility,
it simply adds an extra pair of properties to the decoded object.
If people have not used any headers they will not have a headers key in the `decoded`...
I don't see the disadvantage to setting this to `true`...